### PR TITLE
feat: add /car and /car/id

### PR DESCRIPTION
**Pull request by alex-gudoitsev**
- Add lambda function /car
- Add api /car in FE and show the list of cars
- Add lambda function /car/{id}

FE link: https://dc4nabuctzdtz.cloudfront.net/

API:
https://o11h5923hh.execute-api.us-east-1.amazonaws.com/dev/cars
https://o11h5923hh.execute-api.us-east-1.amazonaws.com/dev/cars/1

image

### CODE REVIEW

**Comment by VittalQSC** [comment]
**What was done**

Tasks: 
+4 Your own Frontend application is integrated with Product Service (/products API) and products from Product Service are represented on Frontend. AND POINT1 and POINT2 are done

Additional:
+1 async/await
+1 ES6 Modules
+1 ESBuild bundler 
0 swagger (correct me if im wrong)
0 unit tests
+1 code splitting
0 main error scenarios handled

total score: 8

File: product-service-ts/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "json-schema-to-ts": "^1.5.0",
+    "serverless": "^3.0.0",
+    "serverless-esbuild": "^1.23.3",
+    "ts-node": "^10.4.0",
+    "tsconfig-paths": "^3.9.0",
+    "typescript": "^4.1.3"
+  },
+  "author": "The serverless webpack authors (https://github.com/elastic-coders/serverless-webpack)",
+  "license": "MIT"
+}
```
Comment (SlavaJSFE) [inline product-service-ts/package.json:30]: Missing blank line

File: product-service-ts/src/functions/carsId/staticData.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { ICar } from './types';
+
+export const staticData: ICar[] = [
```
Comment (SlavaJSFE) [inline product-service-ts/src/functions/carsId/staticData.ts:3]: Code duplication. You already have the same data.

**Review by SlavaJSFE — APPROVED** [review]

**Comment by SlavaJSFE** [comment]
Main part evaluation: 4 points
Additional points:
+1 - ES6 modules are used for Product Service implementation
+1 (All languages) - Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file) and separated in codebase.
About ESBuild - it's "Not applicable for preconfigured/built-in bundlers that come with templates, plugins, etc." Existing configuration comes with serverless template.

#### Final mark 6

File: product-service-ts/src/functions/carsId/staticData.ts (hunk 2 above)
Comment (alex-gudoitsev) [reply product-service-ts/src/functions/carsId/staticData.ts:3]: Ok, will remove

**Pull request merged**